packages/core/src/types.ts

```diff
@@ -94,8 +106,7 @@ export const aggregateByList = [
 ] as const;
 export type AggregateBy = typeof aggregateByList[number];
 
-export const sortByList = ['Degree', 'Size', 'Deviation'] as const;
-export type SortBy = typeof sortByList[number];
+export type SortBy = string;
```

Comment:
Is it worth still having a `SortBy` type when it's just an alias for `string`? Seems like it'd be more clear to use `string` so that devs don't have to look up the `SortBy' type to understand what it is.

Reply:
This is resolved in #358